libs/blocks/locui-create/fragments/view.js

```diff
@@ -56,7 +56,9 @@ export default function FragmentsSection({
    </ul>`
 }
    <div>
-    ${errorFragments && errorFragments.length > 0 && html`<div class='form-field-error'>Invalid fragments <a href="#" onClick=${() => showFragments(errorFragments)}>view details</a></div>`}
+    ${errorFragments && errorFragments.length > 0 && html`<div class='form-field-error'>Invalid fragments <button class='locui-invalid-fragments-btn' onClick=${() => {
+    showFragments(errorFragments);
+  }}>view details</button></div>`}
```

Comment:
Nit: You can shorten this with optional chaining.
```suggestion
    ${errorFragments?.length > 0 && html`<div class='form-field-error'>Invalid fragments <button class='locui-invalid-fragments-btn' onClick=${() => {
    showFragments(errorFragments);
  }}>view details</button></div>`}
```

Reply:
Hi @robert-bogos ,
Thanks , fixed the review comment.